src/zero/api/handlers.clj

```diff
@@ -101,16 +102,23 @@
   [request]
   (let [uuids (-> request :parameters :body distinct)]
     (letfn [(q [[left right]] (fn [it] (str left it right)))]
-      (jdbc/with-db-transaction [tx (postgres/zero-db-config)]
-        (->> uuids
-             (map :uuid)
-             (map (q "''")) (str/join ",") ((q "()"))
-             (format "SELECT * FROM workload WHERE uuid in %s")
-             (jdbc/query tx)
-             (run! (partial start-workload! tx)))
-        (->> uuids
-             (mapv (partial postgres/get-workload-for-uuid tx))
-             succeed)))))
+      (let [db-config (postgres/zero-db-config)
+            db-conn (jdbc/get-connection db-config)
+            query (->> (repeat (count uuids) "?")
+                       (str/join ",") ((q "()"))
+                       (format "SELECT * FROM workload WHERE uuid in %s"))
+            ps (jdbc/prepare-statement db-conn query)]
+        (doseq [[i uuid] (map-indexed vector uuids)] (.setString ps (+ i 1) (:uuid uuid)))
+        (jdbc/with-db-transaction [tx db-config]
+          (try
+            (->> ps
+               (jdbc/query tx)
+               (run! (partial start-workload! tx)))
+            (->> uuids
+               (mapv (partial postgres/get-workload-for-uuid tx))
+               succeed)
+            (catch Exception e
```

Comment:
I think there's error handling further up the stack in the http request handling library that reponds to a request with the exception message.
As a side note, we'll probably do something like this later when we add logging

Reply:
You're right! The default exception handler in the exception-middleware will catch this and throw a 500 error so I removed the try/catch